timers.js

```diff
@@ -1,4 +1,31 @@
-
-
-
-
+// Step 1.
+var resetButton = document.querySelector("#reset");
+var startButton = document.querySelector("#start") ;
+var pauseButton = document.querySelector("#pause");
+
+// Step 2. & Step 5,6,7
+
+resetButton.addEventListener("click");
+
+startButton.addEventListener("click", function() {
+	watchTimer = setInterval(updateTime, 1000);
+});
+
+pauseButton.addEventListener("click", function() {
+	watchTimer = clearInterval(updateTime, 10000);
+});
+
+// Step 3.
+
+var seconds;
+var timerId;

```

Comment:
With you `setInterval` saved in `timerId`, you can use this same variable to stop/reset the timer as well using `clearInterval`.